### PR TITLE
16841 review stripe package

### PR DESCRIPTION
**Pull request by jsanchez-slingr**
Pull-request for issue [#16841](https://github.com/slingr-stack/platform/issues/16841) created by @jsanchez-slingr
## What it does?
Review Stripe package
## How to test it?
You can use https://eventplanner.idea2.io/dev/builder/#model/packages or register on tarzan https://github.com/slingr-stack/platform/wiki/How-To-Commons#packages
Execute the scripts from the readme.
[Checklist for packages](https://docs.google.com/document/d/1rMDvwVnbCTNuC0Z8z3An3c5UpnvY4S2CUEuJMoLsAAg/edit#heading=h.tn1fsa44no8t)
Remember to use this [checklist](https://docs.google.com/document/d/139rKhIsvpT3yBSB88BBZfy03fD_Sried0bVwIWVaHJk/edit?usp=sharing) for the review.
## Technical notes
There are no technical notes.
## Deployment notes
There are no deployment notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,12 +10,14 @@ The Stripe package has the following features:
...
-In most cases, you will be using the provided shortcuts to access the API. For example, you could use the API directly by doing an HTTP request like this:
+In most cases, you will use the provided shortcuts to access the API. For example, you could use the API directly by making an HTTP request like this:
 
 ```js
 let res = pkg.stripe.api.get('/v1/customers');
 ```
 
+For further details about the Stripe API, please take a look at the [documentation.](https://docs.stripe.com/api)
+
 # Configuration
```
Comment (pasaperez-work) [inline README.md]: Let's use the same format for all configuration parameters, check the mandrill's readme format: https://github.com/slingr-stack/mandrill-package/?tab=readme-ov-file#api-key

File: README.md [hunk 2]
```diff
@@ -61,15 +63,15 @@ let customers = pkg.stripe.api.get('/v1/customers');
 let customersByEmail = pkg.stripe.api.get('/v1/customers?email=test@example.com');
```
Comment (pasaperez-work) [inline README.md]: Please add a couple more of simple call examples on the JS API.

File: README.md [hunk 3]
```diff
@@ -61,15 +63,15 @@ let customers = pkg.stripe.api.get('/v1/customers');
...
+The package automatically handles authentication, so there’s no need to worry about that.
 
 For more information about making HTTP calls, please refer to the documentation of the [HTTP service](https://github.com/slingr-stack/http-service).
 
 # Events
 
 ## Webhook
 
-Every time Stripe sends an event to the webhooks URL (it needs to be configured in Stripe), this event will be triggered. The package takes care of verifying the signature if needed. The event's data is the raw content sent by Stripe. Check the [webhooks documentation](https://docs.stripe.com/api/events) for more information.
+Every time Stripe sends an event to the webhook URL (which needs to be configured in Stripe), this event will be triggered. The package takes care of verifying the signature if needed. The event's data is the raw content sent by Stripe. Check the [webhooks](https://docs.stripe.com/api/events) documentation for more information.
```
Comment (pasaperez-work) [inline README.md]: Add this link to webhooks tests: https://docs.stripe.com/webhooks

File: scripts/api.js [hunk 4]
```diff
@@ -119,34 +81,6 @@ exports.delete = function (path, httpOptions, callbackData, callbacks) {
...
- *
- * @param {string} path         - The path to send the HEAD request to.
- * @param {object} httpOptions  - The options to be included in the HEAD request check http-service documentation.
- * @param {object} callbackData - Additional data to be passed to the callback functions. [optional]
- * @param {object} callbacks    - The callback functions to be called upon completion of the HEAD request. [optional]
- * @return {object}             - The response of the HEAD request.
- */
-exports.head = function (path, httpOptions, callbackData, callbacks) {
-    let options = checkHttpOptions(path, httpOptions);
-    return httpService.head(stripe(options), callbackData, callbacks);
```
Comment (pasaperez-work) [inline scripts/api.js:133]: Refactor `stripe` function to `Stripe`.

File: README.md [hunk 5]
```diff
@@ -24,19 +26,19 @@ First, you will need to set up an account in Stripe. You can easily create a dev
...
+The secret key can be found in the [dashboard](https://dashboard.stripe.com/apikeys) of your Stripe app. Simply copy the generated API secret key into this field.
 
 ## Webhook URL
 
 Name: `webhookUrl`
 
-This is the URL you should configure for webhooks in Stripe. If you don't configure the webhooks URL in Stripe you app won't receive the events.
+This is the URL you should configure for webhooks in Stripe. If you don't configure the webhook URL in Stripe, your app won't receive the events.
 
 ## Check Webhooks Signature
```
Comment (pasaperez-work) [inline README.md]: Check resolved comments on this PR: https://github.com/slingr-stack/google-storage-package/pull/1#discussion_r1790138744
And review the Capital case format for titles.

**Review by pasaperez-work — APPROVED** [review]
For the most part it looks good, I'm adding Jose to do the review from a QA point of view and add to the package coverage.

**Comment by jsanchez-slingr** [comment]
@pasaperez-slingr let me know if now looks good.

**Comment by pasaperez-work** [comment]
> For the most part it looks good, I'm adding Jose to do the review from a QA point of view and add to the package coverage.

@barrancojose

**Comment by pasaperez-work** [comment]
> @pasaperez-slingr let me know if now looks good.

Looks good, thanks!

**Pull request merged**